src/components/tabs/tabs.js

```diff
@@ -296,17 +306,21 @@ export default Vue.extend({
       this.updateTabs()
       // Observe child changes so we can update list of tabs
       this.setObserver(true)
+      // Flag we are now mounted and to switch to DOM for tab probing
+      this.isMounted = true
```

Comment:
Do we still need the this.updateTabs() before the this.setObserver(true) call since this.isMounted = true updates the tabs afterwards anyway?

Reply:
Tests failed without this update using the first rendered default slot values, before switching to the tabs discovered via the DOM.